src/register/EmbeddableRegistrationPage.jsx

```diff
@@ -307,12 +377,21 @@ const EmbeddableRegistrationPage = (props) => {
         payload[fieldName] = configurableFormFields[fieldName];
       }
     });
-
     // Don't send the marketing email opt-in value if the flag is turned off
     if (!flags.showMarketingEmailOptInCheckbox) {
       delete payload.marketingEmailsOptIn;
     }
-
+    let fieldError = false;
+    const dirtyFormFields = {};
+    Object.entries(payload).forEach(([key, value]) => {
+      ({ fieldError } = validateInput(key, value, payload, false, true));
+      if (fieldError !== '') {
+        dirtyFormFields[key] = fieldError;
+      }
+    });
+    if (Object.keys(dirtyFormFields).length > 0) {
```

Comment:
In case of invalid registration, is the focus going to top of form like in registration form?

Reply:
yes